VideoLocker/res/layout/drawer_navigation.xml

```diff
@@ -11,34 +12,51 @@
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
         android:background="@color/grey_name_background"
-        android:orientation="vertical"
-        android:padding="15dp">
+        android:orientation="horizontal"
+        android:padding="@dimen/edx_margin"
+        android:gravity="center">
+
+        <de.hdodenhof.circleimageview.CircleImageView
+            android:id="@+id/profile_image"
+            android:layout_width="70dp"
+            android:layout_height="70dp"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_loading_placeholder"
+            android:layout_marginRight="@dimen/edx_margin"
+            tools:src="@drawable/app_icon"
+            app:civ_border_width="1dp"
+            app:civ_border_color="@color/white" />
 
-        <org.edx.mobile.view.custom.ETextView
-            android:id="@+id/name_tv"
-            style="@style/regular_white_text"
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:gravity="left|start|center_vertical"
-            android:inputType="textCapWords"
-            android:maxWidth="200dp"
-            android:paddingTop="10dp"
-            android:textSize="20sp"
-            tools:text="Username"
-            tools:targetApi="17" />
+            android:background="@color/grey_name_background"
```

Comment:
This background is no longer necessary in this sub-container, as it's already implemented in the parent.